starlite/logging/utils.py

```diff
@@ -2,7 +2,7 @@
 
 
 def resolve_handlers(handlers: List[Any]) -> List[Any]:
-    """Converts list of string of handlers to the object of respective handler.
+    """Convert list of string of handlers to the object of respective handler.
```

Comment:
```suggestion
    """Convert list of string handler names to the object of respective handler.
```
Not sure if my suggestion is correct, but this reads awkward.